torch/_inductor/config.py

```diff
@@ -200,6 +200,18 @@ def is_fbcode():
     "TORCHINDUCTOR_MAX_AUTOTUNE_GEMM_BACKENDS", "ATEN,TRITON"
 ).upper()
 
+# Number of threads to use for pre-compilation of kernels
+# in order to populate compilation cache.
+autotune_precompilation_workers = min(
+    torch.get_num_threads(),
+    int(
+        os.environ.get(
+            "TORCHINDUCTOR_MAX_AUTOTUNE_PRECOMPILATION_WORKERS",
+            str(torch.get_num_threads() - 8),
```

Comment:
Why -8?  This could be negative on some machines.

Reply:
Yes, negative values are allowed, in that case we simply don't have parallel precompilation. The idea is to leave at least 8 CPUs unused by parallel precompilation by default. But we could change that of course.

Reply:
Why not use just use torch.get_num_threads()?

Reply:
The idea is to leave 8 CPUs unused under the assumption that these are required for whatever else is happening on the system. But I will remove the -8.